ecommerce-website/src/store/reducers/ProductDetails.js

```diff
@@ -0,0 +1,40 @@
+const initialState = {
+    cartProducts:[],
+    count:0
+};
+
+const reducer = ( state = initialState, action ) => {
+    switch ( action.type ) {
+        case 'ADD_PRODUCT_TO_CART': 
+            return {
+                ...state,
+                cartProducts:state.cartProducts.concat(action.productdetail),
```

Comment:
Should we do some kind of ID checking? Just to check if that product is already added to the cart?